src/Provider/AbstractProvider.php

```diff
@@ -24,16 +24,6 @@
     const ACCESS_TOKEN_UID = null;
 
     /**
-     * @var string HTTP method used to fetch access tokens.
```

Comment:
HTTP request methods are constants. What if a provider uses `PUT` or some arbitrary `TOKEN` method? These should be normalised when used by the client.